integrations/dd-sdk-android-okhttp/src/main/kotlin/com/datadog/android/okhttp/trace/TracingInterceptor.kt

```diff
@@ -549,7 +553,7 @@ internal constructor(
             setSampledOutHeaders(tracedRequestBuilder, tracingHeaderTypes, span, tracer)
         } else {
             tracer.propagate().inject(
-                span.context(),
+                span.extractRumContext(rumContextPropagator, block = true).context(),
```

Comment:
is this call needed? we will try to get the result of the `Future` for a second time, setting same value again, because we already resolved them in `interceptAndTrace` call up in the stack.

Reply:
Good catch. Removed